mms/export_model.py

```diff
@@ -391,12 +392,21 @@ def export_model(model_name, model_path, service_file=None, export_file=None):
 
         if model_path.startswith('~'):
             model_path = os.path.expanduser(model_path)
-
-        files = os.listdir(model_path)
+        # Entry point model here, this is in the main folder
+        files = set(os.listdir(model_path))
```

Comment:
Looks good... Just one concern, we seem to be overloading files to be a "set" and a "list". Could you make these two variables seperate. Maybe fileSet and fileList or some such. Just to make things cleaner.